.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - "node"
+  - "8"
+before_install: |
+  set -e
+  export PATH="./$HOME/node_modules/.bin:$PATH"
+  date
+install: |
+  set -e
+  date
+  npm install
+  date
+script:  npm run prepublish && npm run test 2>&1
```

Comment:
```suggestion
script:  yarn build && yarn test
```

Reply:
This one too. But i haven't run lint as of now, because, i see there are a lot of lint errors if i run eslint. Should we fix the errors and then include linting in the pipeline

Reply:
Oh sorry, i juest checked your comment about rules and formatiing. will make changes now

Reply:
Done now